supply/lib/supply/uploader.rb

```diff
@@ -168,34 +168,34 @@ def update_track(apk_version_codes)
     #  - Lesser than the greatest of any later (i.e. production) track
     #  - Or lesser than the currently being uploaded if it's in an earlier (i.e. alpha) track
     def check_superseded_tracks(apk_version_codes)
-      UI.message("Checking superseded tracks...")
+      UI.message("Checking superseded tracks, uploading '#{apk_version_codes}' to '#{Supply.config[:track]}'...")
       max_apk_version_code = apk_version_codes.max
       max_tracks_version_code = nil
 
       tracks = ["production", "rollout", "beta", "alpha"]
       config_track_index = tracks.index(Supply.config[:track])
 
       tracks.each_index do |track_index|
-        next if track_index.eql? config_track_index
         track = tracks[track_index]
-
         track_version_codes = client.track_version_codes(track).sort
+        UI.verbose("Found '#{track_version_codes}' on track '#{track}'")
+
+        next if track_index.eql? config_track_index
```

Comment:
This was moved down here so the verbose output would still happen?
What exactly does this check? (Maybe add as comment?)

Reply:
We need verbose output in order to get more info why it does failed, more description is in #9574.